pytorch_lightning/plugins/training_type/parallel.py

```diff
@@ -133,6 +133,11 @@ def block_backward_sync(self):
             yield None
 
     def teardown(self) -> None:
+        # Un-reference the wrapper if any was used.
+        # todo (tchaton): Add support for all plugins.
+        if isinstance(self.model, DistributedDataParallel):
+            self.model = self.lightning_module
```

Comment:
This is the only place with a hard reference to this, right?

Reply:
Yes.